### PR TITLE
TELCODOCS-1433 Docs NRO Must Gather

### PR DESCRIPTION
**Pull request by kquinn1204**
[TELCODOCS-1433]: 

Version(s): 4.14 and main

Issue: https://issues.redhat.com/browse/TELCODOCS-1433

Link to docs preview: 

* https://63481--docspreview.netlify.app/openshift-enterprise/latest/support/gathering-cluster-data.html#gathering-data-specific-features_gathering-cluster-data
*  https://63481--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/cnf-numa-aware-scheduling.html#cnf-about-collecting-nro-data_numa-aware

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by kquinn1204** [comment]
got LGTM via slack

File: modules/gathering-data-specific-features.adoc [hunk 1]
```diff
@@ -63,6 +63,9 @@ endif::openshift-dedicated[]
 |`registry.redhat.io/workload-availability/self-node-remediation-must-gather-rhel8:v<installed-version-SNR>`
 |Data collection for the Self Node Remediation (SNR) Operator and the Node Health Check (NHC) Operator.
 
+|`registry.redhat.io/numaresources/numaresources-must-gather-rhel9:v<installed-version-NRO>`
```
Comment (jeana-redhat) [inline modules/gathering-data-specific-features.adoc:66]: User-replaceable values should be lowercase and underscore-separated. I see there is a mix of formatting in this table already, but would be good to only add new ones following the convention.
```suggestion
|`registry.redhat.io/numaresources/numaresources-must-gather-rhel9:v<installed_version_nro>`
```

File: modules/cnf-about-collecting-nro-data.adoc [hunk 2]
```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/cnf-numa-aware-scheduling.adoc
+
+:_content-type: CONCEPT
```
Comment (jeana-redhat) [inline modules/cnf-about-collecting-nro-data.adoc:5]:
```suggestion
:_content-type: PROCEDURE
```

File: modules/cnf-about-collecting-nro-data.adoc [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+
+* The {product-title} CLI (`oc`) installed.
+
+.Precedure
+
+. To collect NUMA-aware pod scheduling, data with `must-gather`, you must specify the NUMA-aware pod scheduling image.
```
Comment (jeana-redhat) [inline modules/cnf-about-collecting-nro-data.adoc:19]:
```suggestion
* To collect NUMA-aware pod scheduling, data with `must-gather`, you must specify the NUMA-aware pod scheduling image.
```

File: modules/cnf-about-collecting-nro-data.adoc [hunk 4]
```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/cnf-numa-aware-scheduling.adoc
+
+:_content-type: CONCEPT
+[id="cnf-about-collecting-nro-data_{context}"]
+= About collecting NUMA-aware pod scheduling data
```
Comment (jeana-redhat) [inline modules/cnf-about-collecting-nro-data.adoc:7]: The title is throwing me off a bit. Seems like "About" shouldn't be at the end of a section. This is also formatted as a procedure, so how about just starting with the verb?
```suggestion
= Collecting NUMA-aware pod scheduling data
```

File: modules/cnf-about-collecting-nro-data.adoc [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+You can use the `oc adm must-gather` CLI command to collect information about your cluster, including features and objects associated with NUMA-aware pod scheduling.
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+
+* The {product-title} CLI (`oc`) installed.
```
Comment (jeana-redhat) [inline modules/cnf-about-collecting-nro-data.adoc:15]: There is a newish attribute to use for the first instance of `oc` :nerd_face: 
```suggestion
* The {oc-first} installed.
```

**Review by jeana-redhat — COMMENTED** [review]
Just a few things here to look at.

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

File: modules/cnf-about-collecting-nro-data.adoc [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+
+* The {oc-first} installed.
+
+.Procedure
+
+* To collect NUMA-aware pod scheduling, data with `must-gather`, you must specify the NUMA-aware pod scheduling image.
```
Comment (bscott-rh) [inline modules/cnf-about-collecting-nro-data.adoc:19]:
```suggestion
* To collect NUMA-aware pod scheduling data with `must-gather`, you must specify the NUMA-aware pod scheduling image.
```
Extra comma

File: modules/cnf-about-collecting-nro-data.adoc [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+[id="cnf-about-collecting-nro-data_{context}"]
+= Collecting NUMA-aware pod scheduling data
+
+You can use the `oc adm must-gather` CLI command to collect information about your cluster, including features and objects associated with NUMA-aware pod scheduling.
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+
+* The {oc-first} installed.
```
Comment (bscott-rh) [inline modules/cnf-about-collecting-nro-data.adoc:15]:
```suggestion
* You have access to the cluster as a user with the `cluster-admin` role.

* You have installed the {oc-first}.
```
Reworded to make these full sentences

**Review by bscott-rh — COMMENTED** [review]
2 more small nits while merge reviewing. Let me know when these are resolved and I'll merge

**Comment by kquinn1204** [comment]
@bscott-rh Thanks for picking those up, hopefully all sorted now

**Pull request merged**